### PR TITLE
Fix microsalt deliverables path and run status

### PR DESCRIPTION
**Pull request by barrystokman**
This PR adds/fixes `cg workflow microsalt store completed`: it now uses the correct path to the deliverables files

### How to prepare for test
- [x] ssh to hasta
- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh fix/microsalt-deliverables-path`

### How to test
- [x] login to hasta
- [x] do `us`
- [x] set test preconditions:
  - [x] delete any existing bundles for 609108/merrymink: `housekeeper delete version -b 609108`
  - [x] delete any existing analyses for 609108/merrymink
  - [x] set `action` of families 609108/merrymink and 551129/vitalmidge to `running`
- [x] run `cg workflow microsalt store completed`

### Expected test outcome:
- [x] a new bundle and version have been created for 609108: `new bundle added: 609108, version 2020-10-19`
- [x] a new bundle version directory has been created: `/home/proj/stage/housekeeper-bundles/609108/2020-10-19`
- [x] files in the deliverables file have been stored in Housekeeper
- [x] an analysis object has been created for 609108/merrymink
- [x] `action` of case merrymink has been reset
- [x] Take a screenshot and attach or copy/paste the output

## Review
- [x] code approved by MR
- [x] tests executed by @barrystokman 
- [x] "Merge and deploy" approved by MR
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by barrystokman** [comment]
`cg workflow microsalt store completed`
image
:heavy_check_mark: 

Regression test `cg workflow microsalt store analysis`:
image
:heavy_check_mark:

File: cg/cli/workflow/microsalt/base.py [hunk 1]
```diff
@@ -122,6 +122,8 @@ def config_case(context: click.Context, dry_run: bool, ticket: int, sample_id: s
 def run(context: click.Context, dry_run: bool, config_case_path: click.Path, ticket: int):
     """ Start microSALT with an order_id """
 
+    microbial_api = context.obj["analysis_api"]
```
Comment (Mropat) [inline cg/cli/workflow/microsalt/base.py:125]: Could you re-name this object to be more microsalt-specific? Such as microsalt_analysis_api, to keep it more distinct form other analysis api we have in cg

**Review by Mropat — APPROVED** [review]
:tada:

**Comment by barrystokman** [comment]
Bumped:
image

Deployed:
image

**Pull request merged**